src/client/state/cons.js

```diff
@@ -91,6 +92,7 @@ const cons = {
       REPLY_TO_CLICKED: 'REPLY_TO_CLICKED',
       SEARCH_OPENED: 'SEARCH_OPENED',
       REUSABLE_CONTEXT_MENU_OPENED: 'REUSABLE_CONTEXT_MENU_OPENED',
+      OPEN_NAVIGATION: 'OPEN_NAVIGATION',
```

Comment:
we should rename this to NAVIGATION_OPENED cuz it's an event that happened and we are notifying about it to UI.